frontend/src/containers/Search.tsx

```diff
@@ -324,6 +325,31 @@ function Search() {
         </div>
       </div>
       <div className="search-result-container">
+        {searched == false && (
+          <>
+            <div className="search-result-header">
+              <div className="search-result-title">Recent posts</div>
+            </div>
+            <div className="search-result-content">
+              {true &&
+                postsAll.map((post: SimplePostType) => {
```

Comment:
Is all of the shared posts rendered on the main page? I think we should limit the number of posts we show for the first time (about 5?)